### DISCUSSION TITLE
How does one access the current user on the backend? (e.g. nodejs)

### DISCUSSION THREAD

**Discussion by maximedupre** [supabase/supabase#6139] [Questions]
There doesn't seem to be any official documentation on this subject, except for [this mention](https://supabase.com/docs/reference/javascript/auth-user):

> For server-side management, you can get a user through auth.api.getUserByCookie()

But it does not explain how to use it or link to some documentation.

I only have the Twitter auth activated for my site.

How does one access the current user on the backend?

# Edit 1

I guess this is another way of getting the current user on the backend: https://github.com/supabase/supabase/issues/491#issuecomment-871863896

**Comment by Martin-Hausleitner** [comment]
You can write a business logic with https://supabase.com/docs/guides/database/functions
